nixos/modules/virtualisation/google-compute-config.nix

```diff
@@ -2,13 +2,55 @@
 with lib;
 let
   gce = pkgs.google-compute-engine;
+  agent = pkgs.google-compute-guest-agent;
+  guest-configs = pkgs.google-compute-configs;
 in
 {
   imports = [
     ../profiles/headless.nix
     ../profiles/qemu-guest.nix
   ];
 
+  environment.etc."sysctl.d/11-gce-network-security.conf".source = "${gce}/sysctl.d/11-gce-network-security.conf";
+
+  environment.etc."default/instance_configs.cfg".text = ''
+    [Accounts]
```

Comment:
@NinjaTrappeur can you compare your configuration here to the one from my PR and tell which other options are a good idea to use?